Dockerfile.development

```diff
@@ -0,0 +1,42 @@
+FROM ruby:3.3.0
+
+RUN apt-get update -yq \
+  && apt-get upgrade -yq \
+  #ESSENTIALS
+  && apt-get install -y -qq --no-install-recommends build-essential curl git-core vim passwd unzip cron gcc wget netcat-traditional \
+  # RAILS PACKAGES NEEDED
+  && apt-get update \
+  && apt-get install -y --no-install-recommends imagemagick postgresql-client libvips \
+  # INSTALL NODE
+  && curl -sL https://deb.nodesource.com/setup_16.x | bash - \
+  && apt-get install -y --no-install-recommends nodejs \
```

Comment:
do we really need nodejs?
and since we use 20 in tools version, why would we have 16 here?